### PR TITLE
10245 manual increment

### PR DESCRIPTION
**Pull request by smoyth**

### CODE REVIEW

File: db/migrate/20191122175425_remove_current_version_id_from_forms.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class RemoveCurrentVersionIdFromForms < ActiveRecord::Migration[5.2]
+  def change
+    remove_column(:forms, :current_version_id)
```
Comment (cooperka) [inline db/migrate/20191122175425_remove_current_version_id_from_forms.rb]: Please also add type so I can undo when changing branches :)

File: app/models/form_version.rb [hunk 2]
```diff
@@ -35,10 +35,6 @@ class FormVersion < ApplicationRecord
   after_initialize :generate_code, :generate_number
   before_create :ensure_unique_code, :ensure_unique_number
 
-  # Only one can be true per form.
-  validates :current, uniqueness: {allow_blank: true, scope: :form_id}
-  validates :minimum, uniqueness: {allow_blank: true, scope: :form_id}
```
[the remark this answers is not in the document]
Comment (smoyth) [reply app/models/form_version.rb]: See the unique indices on the table. You asking this made me realize that the `number` one had to be scoped to form, so thanks! I just made that change.

File: config/locales/en/main.yml [hunk 3]
```diff
@@ -608,7 +609,7 @@ en:
         recipient_ids: "If you enter user or user group names here, any incoming SMS responses will be forwarded to them automatically."
         authenticate_sms: "Whether to authenticate SMS responses. Disabling this can allow anyone who knows the phone number of an enumerator to spoof submissions."
         allow_incomplete: "If checked, allows forms with incomplete required questions to be submitted anyway (via web and ODK only). Enumerators using ODK must enter an override code (see the Settings page) to do so."
-        version: "The current version number and letter code."
+        current_version_name: "Each form version has a 10-digit ID and a 3-letter code. The form versioning system is intended to indicate small changes in the form design so that observer's can update their copies of the form."
```
[the remark this answers is not in the document]
Comment (smoyth) [reply config/locales/en/main.yml]: Enumerators even!

File: app/views/forms/form.html.erb [hunk 4]
```diff
@@ -21,9 +21,11 @@
 
       <%= f.field(:name, required: true, read_only: cannot?(:rename, form)) %>
       <%= f.field(:status, content: form.status_with_icon) %>
-      <%= f.regenerable_field(:current_version_name,
-            action: :increment_version, link_i18n_key: "form.increment_version",
-            confirm: t("form.increment_version_warning")) %>
+      <% unless form.current_version.nil? %>
```
[the remark this answers is not in the document]
Comment (smoyth) [reply app/views/forms/form.html.erb]: The double negative can be confusing for sure. And I believe this is actually a Rubocop rule but Rubocop doesn't know how to lint erb files. I'll fix.

File: db/migrate/20191122175425_remove_current_version_id_from_forms.rb (hunk 1 above)
Comment (smoyth) [reply db/migrate/20191122175425_remove_current_version_id_from_forms.rb]: Done.

**Comment by smoyth** [comment]
@cooperka Thanks for catching that regression! Fixed now.

**Pull request merged**